index.html

```diff
@@ -0,0 +1,290 @@
+<!--
```

Comment:
Nit, it looks like this came from https://github.com/googleplus/gplus-quickstart-python/blob/master/templates/index.html. It's good practice to mention that, so you can more easily find it in the future.

Have you made any changes to the original file?